protocol/x/clob/keeper/process_operations.go

```diff
@@ -248,6 +219,229 @@ func (k Keeper) PersistMatchToState(
 	return nil
 }
 
+// statUnverifiedOrderRemoval stats the unverified order removal along with the block proposer. This
+// is used to track the number of unverified order removals a proposer has made, which may indicate malicious
+// behavior.
+func (k Keeper) statUnverifiedOrderRemoval(ctx sdk.Context, orderRemoval types.OrderRemoval) {
```

Comment:
ahhh I see you added all these logic to get the moniker (adding staking keeper which requires updates to all unit tests, etc). 

initially I thought we can just use the cons address as labels (just need to do something like `sdk.ConsAddress(block.Header().Propoer).String()`. Should be trivial to find the corresponding moniker manually I think.

Reply:
do you feel like we should leave this in? 

on one hand, it's nice to know the moniker so that it's more human readable. On the other hand, it does add some unnecessary complexity to CLOB (especially for unit tests and end to end tests) and we should generally keep it lean unless needed. Will let you make the call here 

Let's do the cleanup in a separate PR if you decide to revert this

Reply:
crap, yeah I just reverted this and updated the PR